### DISCUSSION TITLE
Show dimmer percentage in dial display

### DISCUSSION THREAD

**Discussion by hapklaar** [cgiesche/streamdeck-homeassistant#228] [Feature Requests]
I use a rotary encoder/dial on my Stream Deck+ to control a Zigbee dimmer, which in turn controls an IR panel heater. This works fine, but it doesn't seem to provide a display for the current level of brightness (it's a light entity in HA).

Ideally it would look the same as the built-in Stream dial percentage display.

Maybe this is already possible, but I wasn't able to figure it out. Or if not, could this be added?

**Comment by TonightAt11** [comment]
To show the brightness as a percentage, try adding the line below to your "Custom labels" 

`{{ ((100 / 255) * brightness) | round(0) }}%`

**Reply by hapklaar** [reply]
Thanks, that's already much better

Could be improved even more by using the native UI, like brightness here. Not sure if that's even possibble.

![image](https://github.com/cgiesche/streamdeck-homeassistant/assets/7703320/dd56a420-0923-4ddb-af01-63ddadc32d12)
